wrappers/openvino/rs-vino/openvino-helpers.h

```diff
@@ -185,11 +190,13 @@ namespace openvino_helpers
             openvino_helpers::error_listener error_listener;
             engine.SetLogCallback( error_listener );
     */
+#ifdef OPENVINO2019
```

Comment:
Is there a substitute for newer OpenVino?

Reply:
IErrorListener is deprecated. use exception or return code instead.